### PR TITLE
Galleries

### PR DESCRIPTION
**Pull request by SvLowman**

### CODE REVIEW

File: client/src/pages/GalleryPage.js [hunk 1]
```diff
@@ -1,10 +1,30 @@
...
+const ImageDisplay = styled.div`
+  border: solid 1px lightgray;
+  display: flex;
+  justify-content: center;
+`;
+// const Display = styled.div`
+//   border: solid 1px lightgray;
+//   display: flex;
+//   justify-content: center;
+// `;
```
Comment (marwinburesch) [inline client/src/pages/GalleryPage.js:17]: Please remove unused code :)

File: client/src/pages/GalleryPage.js [hunk 2]
```diff
@@ -18,22 +38,88 @@ const GalleryPage = () => {
...
+  // if (userData && currentImage) {
+  //   console.log(userData.images.indexOf(currentImage));
+  //   const indexOfCurrentImage = userData.images.indexOf(currentImage);
+  //   console.log(indexOfCurrentImage + 1);
+  // }
+
+  // console.log(indexOfCurrentImage);
+
+  // const testArray = ["a", "b", "c"];
+  // console.log(testArray.indexOf("b"));
```
Comment (marwinburesch) [inline client/src/pages/GalleryPage.js:67]: 🧐

File: client/src/pages/GalleryPage.js [hunk 3]
```diff
@@ -1,10 +1,30 @@
...
+  justify-content: center;
+`;
+// const Display = styled.div`
+//   border: solid 1px lightgray;
+//   display: flex;
+//   justify-content: center;
+// `;
+const ImageContainer = styled.div`
+  height: 800px;
+  width: 800px;
```
Comment (marwinburesch) [inline client/src/pages/GalleryPage.js:20]: Are you sure these fixed sizes are necessary? I would expect the parent element to have a size, not determined by its content, but rather a flexible size, which is then passed to its child elements.

Also looking at the implementation, this container has several child elements, an image and two buttons. Wouldn't it make sense to have this as the flexbox and replace `ImageDisplay`?

File: client/src/pages/GalleryPage.js [hunk 4]
```diff
@@ -18,22 +38,88 @@ const GalleryPage = () => {
...
+
+  const nextPlease = () => {
+    const indexOfCurrentImage = userData.images.indexOf(currentImage);
+    setCurrentImage(userData.images[indexOfCurrentImage + 1]);
+  };
+
+  const previousPlease = () => {
+    const indexOfCurrentImage = userData.images.indexOf(currentImage);
+    setCurrentImage(userData.images[indexOfCurrentImage - 1]);
+  };
```
Comment (marwinburesch) [inline client/src/pages/GalleryPage.js:77]: Although it is readable, I think you should consider renaming these two variables. Maybe `nextImage` and `previousImage`?

File: client/src/pages/TaggingPage.js [hunk 5]
```diff
@@ -1,21 +1,69 @@
...
 const Display = styled.div`
   border: solid 1px lightgray;
+  display: flex;
+  justify-content: center;
+`;
+const ImageContainer = styled.div`
+  height: 800px;
+  width: 800px;
+  border: solid 1px lightgray;
+`;
```
Comment (marwinburesch) [inline client/src/pages/TaggingPage.js:18]: These two look familiar, could be separate components don't you think?

File: client/src/pages/TaggingPage.js [hunk 6]
```diff
@@ -1,21 +1,69 @@
...
+const Thumbnail = styled.img`
+  width: 10vw;
+  margin: 0 0.5rem;
 `;
 const TagForm = styled.form`
   border: solid 1px lightgray;
 `;
+const TagNotifier = styled.div`
+  border: solid 1px lightgray;
+`;
```
Comment (marwinburesch) [inline client/src/pages/TaggingPage.js:34]: DRY ;)

File: client/src/pages/TaggingPage.js [hunk 7]
```diff
@@ -1,21 +1,69 @@
...
+  useEffect(() => {
+    doFetch();
+  }, []);
+
+  if (userData) {
+    // console.log(userData.images[userData.images.length - 1].url);
+    // const selectedImage = userData.images[userData.images.length - 1];
+    // console.log(selectedImage.url);
+    // console.log(selectedImage.tags);
+  }
```
Comment (marwinburesch) [inline client/src/pages/TaggingPage.js:51]: This is a wee bit confusing mate

**Review by marwinburesch — CHANGES_REQUESTED** [review]
Looks good! Just a little CSS remark and some DRY comments 🐜 
One thing though: There are a lot of logs and unused code still in here mate, please remove :)

File: client/src/pages/GalleryPage.js (hunk 3 above)
Comment (SvLowman) [reply client/src/pages/GalleryPage.js:20]: Thanks! I'm still working on a good solution for the image display.

File: client/src/pages/TaggingPage.js (hunk 6 above)
Comment (SvLowman) [reply client/src/pages/TaggingPage.js:34]: Thanks! I will put this into a variable when working on the style.

**Comment by SvLowman** [comment]
Thanks for reviewing me!

**Pull request merged**